src/widgets/editor.rs

```diff
@@ -234,7 +234,7 @@ impl Editor {
 impl Widget<EditorState> for Editor {
     fn paint(&mut self, ctx: &mut PaintCtx, data: &EditorState, env: &Env) {
         let rect = (CANVAS_SIZE * data.session.viewport.zoom).to_rect();
-        ctx.fill(rect, &Color::WHITE);
+        ctx.fill(rect, &Color::grey8(0xF0));
```

Comment:
let's do `env.get(theme::GLYPH_LIST_BACKGROUND)`? Assuming these colors are intended to match..

Reply:
I force pushed a new update (4d781ac) that fixes this. Thanks! Please feel free to make any needed changes before merging.